### PR TITLE
refactor(autoware_grid_map_utils): prefix folder structure with autoware/

### PR DESCRIPTION
**Pull request by esteve**
## Description

prefix folder structure with autoware/

## Related links

**Parent Issue:**

- https://github.com/autowarefoundation/autoware/issues/5077
- https://github.com/autowarefoundation/autoware/issues/4569

## Notes for reviewers

None.

## Interface changes

None.

## Effects on system behavior

None.

### CODE REVIEW

**Review by takayuki5168 — APPROVED** [review]
LGTM for planning

**Review by maxime-clem — APPROVED** [review]

**Comment by esteve** [comment]
@maxime-clem @takayuki5168 thank you for your reviews.

@xmfcx @mitsudome-r could you merge this PR when you have a moment? @maxime-clem is a code owner of this package and he approved the changes, so it should be find to merge. Thanks.

**Comment by esteve** [comment]
@xmfcx thanks!

**Pull request merged**
